umi_tools/count_tab.py

```diff
@@ -86,12 +86,17 @@ def main(argv=None):
 
     if argv is None:
         argv = sys.argv
-
+        
```

Comment:
Another tab to remove?

Reply:
There are a few of these in the PR. One in each tool it looks like. Unless I'm missunderstanding the github diff markup.

Reply:
I don't think you are misunderstanding.